crates/astria-sequencer/src/fees/storage/keys.rs

```diff
@@ -12,7 +23,31 @@ pub(in crate::fees) const FEE_CHANGE: &str = "fees/fee_change";
 pub(in crate::fees) const IBC_RELAYER_CHANGE: &str = "fees/ibc_relayer_change";
 pub(in crate::fees) const SUDO_ADDRESS_CHANGE: &str = "fees/sudo_address_change";
 pub(in crate::fees) const IBC_SUDO_CHANGE: &str = "fees/ibc_sudo_change";
-pub(in crate::fees) const BLOCK: &str = "fees/block_fees";
+pub(in crate::fees) const BLOCK: &str = "fees/block"; // NOTE: `BLOCK` is only used in the ephemeral store.
+pub(in crate::fees) const FEE_ASSET_PREFIX: &str = "fees/fee_asset/";
```

Comment:
Let's rename this to a more explicit (no need to duplicate `fee` everywhere):
```suggestion
pub(in crate::fees) const ALLOWED_ASSET_PREFIX: &str = "fees/allowed_asset/";
```